### PR TITLE
feat: [AXIMST-538] Add errors handling 4xx, 5xx

### PR DESCRIPTION
**Pull request by ruzniaievdm**
[Add-errors-handling-4xx-5xx](https://youtrack.raccoongang.com/issue/AXIMST-538/Add-errors-handling-4xx-5xx)

### CODE REVIEW

File: src/generic/saving-error-notification/SavingErrorNotification.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react';
```
Comment (PKulkoRaccoonGang) [inline src/generic/saving-error-notification/SavingErrorNotification.jsx:1]: [question]: Do we need to import the entire react library in this file?

File: src/group-configurations/index.jsx [hunk 2]
```diff
@@ -86,7 +88,9 @@ const GroupConfigurations = ({ courseId }) => {
               {shouldShowExperimentGroups && (
                 <ExperimentConfigurationsSection
                   availableGroups={experimentGroupConfigurations}
-                  experimentConfigurationActions={experimentConfigurationActions}
+                  experimentConfigurationActions={
+                    experimentConfigurationActions
+                  }
```
Comment (PKulkoRaccoonGang) [inline src/group-configurations/index.jsx:93]: [clarify]: Why did you change the formatting?)

```suggestion
                  experimentConfigurationActions={experimentConfigurationActions}
```

**Review by PKulkoRaccoonGang — CHANGES_REQUESTED** [review]

File: src/generic/saving-error-notification/SavingErrorNotification.jsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      )}
+      aria-describedby={intl.formatMessage(
+        messages.warningDescriptionAriaDescribedBy,
+      )}
+    />
+  );
+};
+
+SavingErrorNotification.defaultProps = {
+  errorMessage: '',
```
[the remark this answers is not in the document]
Comment (ruzniaievdm) [reply src/generic/saving-error-notification/SavingErrorNotification.jsx:59]: updated

File: src/group-configurations/index.jsx (hunk 2 above)
Comment (ruzniaievdm) [reply src/group-configurations/index.jsx:93]: ok, leave it as it was

File: src/i18n/messages/ar.json
[the remark this answers is not in the document]
Comment (ruzniaievdm) [reply src/i18n/messages/ar.json:1]: sure, thanks

**Review by PKulkoRaccoonGang — APPROVED** [review]

File: src/certificates/layout/MainLayout.jsx [hunk 4]
```diff
@@ -2,6 +2,7 @@ import PropTypes from 'prop-types';
 import { Container, Layout } from '@openedx/paragon';
 import { useIntl } from '@edx/frontend-platform/i18n';
 
+import { SavingErrorNotification } from '../../generic/saving-error-notification';
```
[the remark this answers is not in the document]
Comment (ruzniaievdm) [reply src/certificates/layout/MainLayout.jsx:5]: renamed

File: src/generic/saving-error-notification/SavingErrorNotification.jsx [hunk 5]
```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Warning as WarningIcon } from '@openedx/paragon/icons';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import { RequestStatus } from '../../data/constants';
+import AlertMessage from '../alert-message';
+import messages from './messages';
+
+const SavingErrorNotification = ({
```
[the remark this answers is not in the document]
Comment (ruzniaievdm) [reply src/generic/saving-error-notification/SavingErrorNotification.jsx:10]: Yeap, it used only for new scope

File: src/certificates/layout/MainLayout.jsx [hunk 6]
```diff
@@ -54,6 +57,10 @@ const MainLayout = ({ courseId, showHeaderButtons, children }) => {
           isShow={isShowProcessingNotification}
           title={processingNotificationTitle}
         />
+        <SavingErrorNotification
+          savingStatus={savingStatus}
+          errorMessage={errorMessage}
+        />
         <InternetConnectionAlert
```
[the remark this answers is not in the document]
Comment (ruzniaievdm) [reply src/certificates/layout/MainLayout.jsx:64]: good catch, thanks
